dbt/tests/generic/test_res_class_matches_pardat.sql

```diff
@@ -86,12 +95,16 @@
         -- Pardat should be unique by (parid, taxyr), so we can select the
         -- max rather than array_agg
         max(pardat.class) as pardat_class
-    from filtered_model
-    left join
+    from filtered_model {{ join_type }}
+    join
         (
             select *
             from {{ source("iasworld", "pardat") }}
-            where cur = 'Y' and deactivat is null
+            where
+                cur = 'Y' and deactivat is null
+                {% if additional_join_condition %}
+                    {{ additional_join_condition }}
+                {% endif %}
```

Comment:
FWIW I interpreted `additional_join_conditions` to mean they would be placed within the join itself i.e. `ON {{ additional_join_conditions`. I don't _think_ that would change the query outcome, though it might change how the planner works. Probably fine either way.

Reply:
Ah, good point. Renamed to `additional_pardat_filter` in 9bf577e63dd08268e869f493b8c42cfaac63cd32 for clarity.